pkg/quick/quick_test.go

```diff
@@ -189,28 +238,43 @@ directories:
 
 	// Save format using
 	config, err := New(&saveMe)
-	c.Assert(err, IsNil)
-	c.Assert(config, Not(IsNil))
+	if err != nil {
+		t.Fatal(err)
+	}
```

Comment:
check for config itself not being null, has been removed. Is it OK, if we check just for no error case?

Reply:
That is enough in this scenario.